CHANGES.rst

```diff
@@ -472,6 +472,8 @@ astropy.io.misc
 - Fix ASDF serialization of circular model inverses, and remove explicit calls
   to ``asdf.yamlutil`` functions that became unnecessary in asdf 2.6.0. [#10189, #10384]
 
+- Resolve deprecation warning from asdf 2.8. [#11738]
```

Comment:
We have switched to towncrier. Please see https://github.com/astropy/astropy/blob/main/docs/changes/README.rst and squash this out. Thanks!

Reply:
p.s. Not even sure if change log is needed in this case.

Reply:
Oh oops, I knew that too.  I'll just remove it, I agree it's not a particularly interesting detail.